internal/daysteps/daysteps.go

```diff
@@ -13,8 +19,65 @@ const (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("неверный формат данных")
+	}
+
+	stepsStr := parts[0]
+	durationStr := parts[1]
+
+	if strings.Contains(stepsStr, " ") {
+		return 0, 0, fmt.Errorf("неверный формат шагов")
+	}
+
+	if strings.Contains(durationStr, " ") {
+		return 0, 0, fmt.Errorf("неверный формат времени")
+	}
+
+	stepsStr = strings.TrimSpace(stepsStr)
+	durationStr = strings.TrimSpace(durationStr)
+
+	steps, err := strconv.Atoi(stepsStr)
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибки
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```